velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -13,43 +13,46 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 #include "velox/exec/tests/utils/OperatorTestBase.h"
 #include "velox/functions/prestosql/window/WindowFunctionsRegistration.h"
 
 namespace facebook::velox::window::test {
 
-/// Common set of window function over clauses using a combination of three
-/// columns.
-static std::vector<std::string> kBasicOverClauses = {
-    "partition by c0 order by c1 desc, c2",
-    "partition by c1 order by c0, c2 desc",
-    "partition by c0 order by c2, c1",
-    "partition by c1 order by c2, c0 desc",
-    // No partition by clause.
-    "order by c0 asc nulls first, c1 desc, c2",
-    "order by c1 asc, c0 desc nulls last, c2 desc",
-    "order by c0 asc, c2 desc, c1 asc nulls last",
-    "order by c2 asc, c1 desc nulls first, c0",
-    // No order by clause.
-    "partition by c0, c1, c2",
-};
+/// Exhaustive set of window function over clauses using a combination of four
+/// columns. Columns c0 and c1 have input data meant to test partitioning
```

Comment:
Would it make sense to rename c0 and c1 columns to key0, key1 to clarify that these are used as keys (partition or sorting keys)?

Reply:
So technically only c0 and c1 are used in the partitioning and sorting. but c2 and c3 were added in order by to enforce a full ordering of the result to ensure deterministic results.

I'll add a comment about this, but given that maybe sticking with c0, c1, c2, c3 is alright I think.